investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/util/ProcessInvestigationDataUtil.java

```diff
@@ -50,12 +50,11 @@ public InvestigationTransformed transformInvestigationData(Investigation investi
         transformObservationIds(investigation.getObservationNotificationIds(), investigationTransformed, objectMapper);
         transformInvestigationConfirmationMethod(investigation.getInvestigationConfirmationMethod(), objectMapper);
         processInvestigationPageCaseAnswer(investigation.getInvestigationCaseAnswer(), investigationTransformed, objectMapper);
-        transformNotifications(investigation.getInvestigationNotifications(), objectMapper);
 
         return investigationTransformed;
     }
 
-    private void transformNotifications(String investigationNotifications, ObjectMapper objectMapper) {
+    public void processNotifications(String investigationNotifications, ObjectMapper objectMapper) {
```

Comment:
Do we have any logging message for Notification processing?

Reply:
Yes, but only if the investigation payload is missing notification data or there was an error processing the notification json. This is consistent with the logging for all other processing in the investigation preprocessing. We can create a story to improve this if needed.

Reply:
Sounds good! We might want to add that in as a future enhancement.